llvm/test/CodeGen/AIE/aie2/ra/tie-subregs-flow-3d.mir

```diff
@@ -5,7 +5,7 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 #
 # (c) Copyright 2023-2024 Advanced Micro Devices, Inc. or its affiliates
-# RUN: llc -O2 -mtriple=aie2 %topdown-single -verify-machineinstrs \
+# RUN: llc -O2 -mtriple=aie2 --issue-limit=1 --aie-bottomup-cycles=0 -verify-machineinstrs \
```

Comment:
Did you forget to revert this after regenerating this test? Or do we stop using these lit substitutions?

Reply:
Absolutely. (First question)